internal/config/audit.go

```diff
@@ -67,8 +67,9 @@ func (c *AuditConfig) validate() error {
 // SinksConfig contains configuration held in structures for the different sinks
 // that we will send audits to.
 type SinksConfig struct {
-	LogFile LogFileSinkConfig `json:"log,omitempty" mapstructure:"log"`
-	Webhook WebhookSinkConfig `json:"webhook,omitempty" mapstructure:"webhook"`
+	LogFile         LogFileSinkConfig     `json:"log,omitempty" mapstructure:"log"`
+	Webhook         WebhookSinkConfig     `json:"webhook,omitempty" mapstructure:"webhook"`
+	WebhookTemplate WebhookTemplateConfig `json:"webhookTemplate,omitempty" mapstructure:"webhook_template"`
```

Comment:
this brings up an interesting challenge/question. I had initially thought that templates would be an extension of the existing webhook config, but i see why you did it this way because you can define multiple templates/urls. it still does seem a bit weird though to have both `webhook` and `webhook_templates` IMO.

maybe instead we still only allow for a single `template` and move it under `webhook` as optional?